netbout-web/src/main/java/com/netbout/dynamo/DyBase.java

```diff
@@ -50,6 +50,7 @@
 @Loggable(Loggable.DEBUG)
 @ToString(of = "region")
 @EqualsAndHashCode(of = "region")
+@SuppressWarnings("PMD.ConstructorOnlyInitializesOrCallOtherConstructors")
```

Comment:
@danff OK, but how person solving next puzzle will know that this suppression should also be fixed?